src/components/frame/LandingPage/ColonyCards/ColonyCard.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import BaseColonyCard from './BaseColonyCard/BaseColonyCard.tsx';
+
+export interface ColonyCardProps {
+  colonyName: string;
+  colonyAvatar: string;
+  membersCount: number;
+}
+
+const displayName = 'frame.LandingPage.ColonyCards';
+
+const MSG = defineMessages({
+  members: {
+    id: `${displayName}.members`,
+    defaultMessage: `{members} Members`,
+  },
+});
+
+export const ColonyCard = ({
+  colonyName,
+  colonyAvatar,
+  membersCount = 0,
+}: ColonyCardProps) => (
+  <BaseColonyCard
+    isClickable
+    avatarPlaceholder={
+      <img
+        src={colonyAvatar}
+        alt="Colony avatar"
+        className="w-8 rounded-full"
+      />
+    }
+  >
+    <p className="flex-1 truncate text-ellipsis text-md font-medium">
+      {colonyName}
+    </p>
+    <p className="max-w-[150px] justify-self-end truncate text-ellipsis text-xs font-normal">
```

Comment:
This should also use the `text-gray-600` class as per Figma designs:

<img width="1127" alt="Screenshot 2024-10-26 at 00 20 27" src="https://github.com/user-attachments/assets/c41cbffe-5e1c-4483-aa75-0388df17d860">